manifests/service.pp

```diff
@@ -4,18 +4,36 @@
 class candlepin::service {
   assert_private()
 
-  service { 'tomcat':
-    ensure     => running,
-    enable     => true,
-    hasstatus  => true,
-    hasrestart => true,
-  }
+	if $candlepin::container {
+		ensure_packages(['podman'])
+
+		file { '/etc/sysconfig/candlepin':
+			ensure  => file,
+			content => template("candlepin/candlepin.sysconfig.erb"),
+			before	=> Service[$candlepin::service_name],
+		} ~>
```

Comment:
Technically not really a dependency between these 2 files. At least not a notify